crates/uv-cache/src/lib.rs

```diff
@@ -206,7 +206,7 @@ impl Cache {
     }
 
     /// Persist a temporary directory to the artifact store.
-    pub fn persist(
+    pub async fn persist(
```

Comment:
Important that this is `async` now because we don't want to block the thread during the backoff phase.